socialite/server/graphql/typeDefs.js

```diff
@@ -61,7 +61,7 @@ module.exports = gql`
     	createdAt: String!
         rating: Int!
         volatility: Float!
-        time_answered: Int!
+        times_answered: Int!
```

Comment:
This change does not alter the code flow at other files, right?

Reply:
No, it does not. I checked.